v03_pipeline/lib/reference_datasets/misc.py

```diff
@@ -31,3 +38,31 @@ def filter_contigs(ht, reference_genome: ReferenceGenome):
     return ht.filter(
         hl.set(reference_genome.standard_contigs).contains(ht.locus.contig),
     )
+
+
+def key_by_locus_alleles(ht: hl.Table, reference_genome: ReferenceGenome) -> hl.Table:
+    chrom = (
+        hl.format('chr%s', ht.chrom)
+        if reference_genome == ReferenceGenome.GRCh38
+        else ht.chrom
+    )
+    locus = hl.locus(chrom, ht.pos, reference_genome.value)
+    alleles = hl.array([ht.ref, ht.alt])
+    ht = ht.transmute(locus=locus, alleles=alleles)
+    return ht.key_by('locus', 'alleles')
+
+
+@contextmanager
+def download_zip_file(url, suffix='.zip'):
+    extracted_filename = url.removesuffix('.zip').split('/')[-1]
```

Comment:
actually, this line was sort-of special cased to `mitimpact`... the file that's extracted from the archive is just `MitImpact_db_3.1.3.txt`.  I think that isn't true for `dbnsfp`?

Reply:
oh no you are right